viewer/src/data/index.ts

```diff
@@ -32,7 +32,7 @@ export interface ButtonData {
 
 export type MapData = GaiaHex[];
 export type HighlightHexData = Map<GaiaHex, { cost?: string }>;
-export type LogPlacement = "top" | "bottom" | "off";
+export type LogPlacement = "top" | "top-full" | "bottom" | "bottom-full" | "off";
```

Comment:
I would keep the `full/recent` setting inside the viewer itself. We may want to switch between the two to check the past info of the game, without needing to go to the sidebar.

The placement is a more permanent setting, so it makes sense to keep it in the sidebar and remove the control in the viewer

Reply:
the problem was that the log keeps changing to "recent" when I replay a log - is there another way to avoid that?

Reply:
changed it back now - and it's still working - so it was probably something else.